### PR TITLE
doc: release notes for octopus & nautilus backports 

### PR DESCRIPTION
**Pull request by theanalyst**
Combining the two into a single pr for ease of reviews

### CODE REVIEW

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+* rbd: librbd: ignore -ENOENT error when disabling object-map (`pr#37814 <https://github.com/ceph/ceph/pull/37814>`_, Jason Dillaman)
+* rbd: rbd-nbd: don't ignore namespace when unmapping by image spec (`pr#37811 <https://github.com/ceph/ceph/pull/37811>`_, Mykola Golub)
+* rgw/rgw_file: Fix the incorrect lru object eviction (`pr#37804 <https://github.com/ceph/ceph/pull/37804>`_, luo rixin)
+* rgw: fix expiration header returned even if there is only one tag in the object the same as the rule (`pr#37806 <https://github.com/ceph/ceph/pull/37806>`_, Or Friedmann)
+* rgw: fix: S3 API KeyCount incorrect return (`pr#37810 <https://github.com/ceph/ceph/pull/37810>`_, 胡玮文)
+* rgw: radosgw-admin should paginate internally when listing bucket (`pr#37802 <https://github.com/ceph/ceph/pull/37802>`_, J. Eric Ivancich)
+* rgw: rgw_file: avoid long-ish delay on shutdown (`pr#37552 <https://github.com/ceph/ceph/pull/37552>`_, Matt Benjamin)
+* rgw: use yum rather than dnf for teuthology testing of rgw-orphan-list (`pr#37805 <https://github.com/ceph/ceph/pull/37805>`_, J. Eric Ivancich)
```
Comment (idryomov) [inline doc/releases/nautilus.rst:35]: The changelog doesn't actually include the replay attack fixes.  It looks like Yuri merged them directly, so there is no PR to reference, but perhaps something like "mon/MonClient: bring back CEPHX_V2 authorizer challenges" should be added?

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -1,3 +1,17 @@
+v15.2.6 Octopus
+===============
+
+This is the 6th backport release in the Octopus series. This releases fixes a
```
Comment (idryomov) [inline doc/releases/octopus.rst:4]: releases -> release

File: doc/releases/octopus.rst [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+security flaw affecting Messenger V2 for Octopus & Nautilus. We recommend users
```
Comment (idryomov) [inline doc/releases/octopus.rst:5]: Similarly, "flaw affecting both Messenger v1 and Messenger v2, among ...".

In octopus and master (but not in nautilus), both msgr1 and msgr2 are affected.

**Review by jdurgin — APPROVED** [review]

File: doc/releases/nautilus.rst [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+=================
+
+This is the 14th backport release in the Nautilus series. This releases fixes a
+security flaw affecting Messenger V2 for Octopus & Nautilus, among other fixes
+across components. We recommend users to update to this release.
+
+Notable Changes
+---------------
+
+* CVE 2020-25660: Fix a regression in Messenger V2 replay attacks
```
Comment (idryomov) [inline doc/releases/nautilus.rst:11]: "CVE 2020-25660: CEPHX_V2 replay attack protection lost (only for Messenger v2)"

File: doc/releases/octopus.rst [hunk 5]
```diff
@@ -1,3 +1,17 @@
...
+to update to this release.
+
+Notable Changes
+---------------
+
+* CVE 2020-25660: Fix a regression in Messenger V2 replay attacks
```
Comment (idryomov) [inline doc/releases/octopus.rst:11]: Similarly, "CVE 2020-25660: CEPHX_V2 replay attack protection lost (both for Messenger v1 and Messenger v2)"

**Review by idryomov — CHANGES_REQUESTED** [review]

**Review by yuriw — APPROVED** [review]

File: doc/releases/nautilus.rst [hunk 6]
```diff
@@ -1,3 +1,42 @@
+v14.2.14 Nautilus
+=================
+
+This is the 14th backport release in the Nautilus series. This release fixes a
+security flaw affecting Messenger V2 for Octopus, among other fixes
```
Comment (idryomov) [inline doc/releases/nautilus.rst:5]:
```suggestion
security flaw affecting Messenger v2, among other fixes
```

File: doc/releases/nautilus.rst [hunk 7]
```diff
@@ -1,3 +1,42 @@
...
+across components. We recommend users to update to this release.
+
+Notable Changes
+---------------
+
+* CVE 2020-25660: CEPHX_V2 replay attack protection lost, for Messenger v1 &
+  Messenger v2 (Ilya Dryomov)
```
Comment (idryomov) [inline doc/releases/nautilus.rst:12]:
```suggestion
* CVE 2020-25660: CEPHX_V2 replay attack protection lost, for Messenger v2 (Ilya Dryomov)
```

File: doc/releases/nautilus.rst [hunk 8]
```diff
@@ -1,3 +1,42 @@
...
+* bluestore: test/objectstore/store_test: kill ExcessiveFragmentation test case (`pr#37824 <https://github.com/ceph/ceph/pull/37824>`_, Igor Fedotov)
+* bluestore: BlockDevice.cc: use pending_aios instead of iovec size as ios num (`pr#37823 <https://github.com/ceph/ceph/pull/37823>`_, weixinwei)
+* bluestore: Support flock retry (`pr#37842 <https://github.com/ceph/ceph/pull/37842>`_, Kefu Chai, wanghongxu)
+* bluestore: attach csum for compressed blobs (`pr#37843 <https://github.com/ceph/ceph/pull/37843>`_, Igor Fedotov)
+* osdc/ObjectCacher: overwrite might cause stray read request callbacks (`pr#37813 <https://github.com/ceph/ceph/pull/37813>`_, Jason Dillaman)
+* mgr: avoid false alarm of MGR_MODULE_ERROR (`pr#38069 <https://github.com/ceph/ceph/pull/38069>`_, Kefu Chai, Sage Weil)
+* mgr: fix race between module load and notify (`pr#37844 <https://github.com/ceph/ceph/pull/37844>`_, Mykola Golub, Patrick Donnelly)
+* mon: set session_timeout when adding to session_map (`pr#37554 <https://github.com/ceph/ceph/pull/37554>`_, Ilya Dryomov)
+* mon/MonClient: bring back CEPHX_V2 authorizer challenges (Iyla Dryomov)
```
Comment (idryomov) [inline doc/releases/nautilus.rst:28]:
```suggestion
* mon/MonClient: bring back CEPHX_V2 authorizer challenges (Ilya Dryomov)
```

File: doc/releases/octopus.rst [hunk 9]
```diff
@@ -1,3 +1,17 @@
+v15.2.6 Octopus
+===============
+
+This is the 6th backport release in the Octopus series. This release fixes a
+security flaw affecting Messenger v1 & v2.. We recommend users
```
Comment (idryomov) [inline doc/releases/octopus.rst:5]:
```suggestion
security flaw affecting Messenger v1 & v2. We recommend users
```

File: doc/releases/octopus.rst [hunk 10]
```diff
@@ -1,3 +1,17 @@
...
+to update to this release.
+
+Notable Changes
+---------------
+
+* CVE 2020-25660: CEPHX_V2 replay attack protection lost, for Messenger v1 &
+  Messenger v2 (Ilya Dryomov)
```
Comment (idryomov) [inline doc/releases/octopus.rst:12]:
```suggestion
* CVE 2020-25660: CEPHX_V2 replay attack protection lost, for Messenger v1 & v2 (Ilya Dryomov)

Changelog
---------

* mon/MonClient: bring back CEPHX_V2 authorizer challenges (Ilya Dryomov)
```

**Review by idryomov — CHANGES_REQUESTED** [review]

**Comment by idryomov** [comment]
@theanalyst Still some confusion around msgr1 vs msgr2 and mention of octopus in the nautilus release note.  I've left suggestions this time to make it easier.

**Comment by idryomov** [comment]
Looks good.  Please squash the last commit (nautilus bits into the nautilus commit and octopus bits into the octopus commit) and I'll merge.

**Comment by idryomov** [comment]
Never mind, I did it myself.

**Review by idryomov — APPROVED** [review]

**Pull request merged**